ksql-engine/src/test/java/io/confluent/ksql/topic/TopicCreateInjectorTest.java

```diff
@@ -283,16 +284,16 @@ public void shouldCreateMissingTopic() {
     // Given:
     givenStatement("CREATE STREAM x WITH (kafka_topic='topic') AS SELECT * FROM SOURCE;");
     when(builder.build()).thenReturn(new TopicProperties("expectedName", 10, (short) 10));
-    assertThat("topic did not exist", !topicClient.isTopicExists("expectedName"));
 
     // When:
     injector.inject(statement, builder);
 
     // Then:
-    assertThat(topicClient.createdTopics(),
-        hasEntry(
-            "expectedName",
-            new FakeTopic("expectedName", 10, (short) 10, TopicCleanupPolicy.DELETE)));
+    topicClient.createTopic(
```

Comment:
should this be a `verify` call?

```suggestion
    verify(topicClient.createTopic(
```

Likewise for others in this test...

Reply:
yes... yes it should